contracts/dapis/proxies/DataFeedProxy.sol

```diff
@@ -0,0 +1,25 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "./interfaces/IDapiProxy.sol";
+
+contract DataFeedProxy is IDapiProxy {
```

Comment:
I wonder if these proxy contracts are good use cases for having a function that calls `selfdestruct()`...it should be safe because these contracts will not hold any funds but it might still need to inherit `onlyOwner`.

Reply:
I wouldn't touch `selfdestruct()` with a 10 feet pole atm https://eips.ethereum.org/EIPS/eip-4758

Reply:
LOL I wasn't aware they are planning on deprecating the opcode 😁 My main motivation to suggest some sort of mechanism to do cleanup if a data feed is ever deprecated. Once we deploy a proxy contract for a data feed then it will be there forever and ever.